test/fixtures/admin-signin.json

```diff
@@ -0,0 +1,29 @@
+{
+  "links": {
+    "self": "https://example.com/session"
+  },
+  "data": {
+    "id": "sessionid123",
+    "type": "session",
+    "relationships": {
+      "account": {
+        "links": {
+          "related": "https://example.com/session/account"
+        },
+        "data": {
+          "id": "abc4567",
+          "type": "account"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "id": "abc4567",
+      "type": "account",
+      "attributes": {
+        "username": "chicken@docs.com"
+      }
+    }
+  ]
+}
```

Comment:
Hey @gr2m, could you take a look at this and tell me wether this fixture makes sense?